artiq/dashboard/moninj.py

```diff
@@ -10,7 +10,7 @@
 from artiq.coredevice.comm_moninj import *
 from artiq.gui.tools import LayoutWidget
 from artiq.gui.flowlayout import FlowLayout
-
+from artiq import __artiq_dir__ as artiq_dir, __version__ as artiq_version
```

Comment:
``__artiq_dir__``? Where is it used?

Reply:
Oh its not used, forgot to delete after copying the code.